CoreScriptsRoot/Modules/VR/LaserPointer.lua

```diff
@@ -375,6 +432,76 @@ do --Helper functions
 	function LaserPointer:shouldWalk()
 		return self:isHeadMounted() or self.testWalkToMode
 	end
+
+	function LaserPointer:getRippleOriginPosition()
+		local humanoid = getLocalHumanoid()
+		if not humanoid then return end
+
+		local rootPart = humanoid.Torso
+		if not rootPart then return end
+
+		local hipHeight = humanoid.HipHeight
+		if humanoid.RigType == Enum.HumanoidRigType.R6 then
+			hipHeight = 3
```

Comment:
So I talked to gameplay team, it sounds like this is going to have to stay. HipHeight isn't necessarily intended for this use, and I'm going to propose an API addition for Humanoid.GroundOffset that'll replace this code here.